config/logging.php

```diff
@@ -119,6 +119,11 @@
         'emergency' => [
             'path' => storage_path('logs/laravel.log'),
         ],
+
+        'sentry' => [
+            'driver' => 'sentry',
+            'level' => 'critical',
```

Comment:
Is it meaningful to also log errors?

Reply:
We could do that - I wanted to start conservative because I didn't know what box of pandora I would unleash with error. Lots of code has not been written with the idea the error() would go into sentry, I don't know if it would be good.
My instinct says no for now, so there is a functional difference between throwing critiical and error, but if you prefer I add the error level also fine.

Reply:
Why would that open a box of Pandora? An `error` is something that prevents the normal behaviour of the application and should be fixed. That would mean publiq-platform often fails without us knowing it 😅

Reply:
Now I see we are talking about Sentry, not slack errors, yeah, it makes sense to log that, I will change it.